tests/hpc/slurm_master_backup_db.pm

```diff
@@ -23,21 +23,14 @@ sub run {
     my $self  = shift;
     my $nodes = get_required_var("CLUSTER_NODES");
     $self->prepare_user_and_group();
+    zypper_call('in nfs-client rpcbind');
```

Comment:
would it make sense also to move `zypper in nfs-client` to mount_nfs?

Reply:
sure, why not actually.

Reply:
changed.